src/v2/System/useFeatureFlag.tsx

```diff
@@ -11,6 +11,15 @@ interface FeatureFlagDetails {
   variant: Variant
 }
 
+interface VariantTrackingProperties {
+  experiment_name: string
+  variant_name: string
+  payload?: string
+  context_owner_type?: OwnerType
```

Comment:
`context_owner_type` is required by the [cohesion schema](https://github.com/artsy/cohesion/blob/85bdd8b2eb4334497f9c9856e99cdd70424c6af6/src/Schema/Events/ExperimentViewed.ts#L38), so should be required here, also.